frontend/src/views/DifferentialExpression/components/Main/components/DeResults/components/DifferentialExpressionResults/index.tsx

```diff
@@ -233,7 +235,15 @@ const DifferentialExpressionResults = ({
           <QueryGroupTags queryGroupsWithNames={queryGroupsWithNames} />
         </FilterTagsWrapper>
       </CellGroupWrapper>
-      {nCellsOverlap > 0 && (
+      {!!errorMessage && (
+        <StyledCallout
+          data-testid={DIFFERENTIAL_EXPRESSION_RESULTS_CALLOUT}
+          intent="negative"
```

Comment:
nit: probably need UX's input on this. But loading result probably doesn't need a red/negative message?

E.g.,

```ts
intent={isLoading ? "info" : "negative"}
```

Reply:
Loading was enver actually meant to show the callout - i just set it to null if isLoading instead.

Reply:
That sounds good thank you!